### PR TITLE
Added autoVote flag in stake command

### LINKED ISSUE #357: Need a quick command to stake and start voting

### PR DESCRIPTION
**Pull request by Yashk767**
# Description

Added a autoVote flag in stake command which enables stakers to start voting immediately after stake. 

Fixes #357

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,18 @@ Example:
...
+If you want to vote just after stake automatically, you can just set `autoVote` flag to true in the stake command as stated below
+
+```
+$ ./razor stake --address <address> --value <value> --autoVote <bool>
+```
+
+Example:
+
+```
+$ ./razor stake --address 0x5a0b54d5dc17e0aadc383d2db43b0a0d3e029c4c --value 2000 --autoVote true
```
Comment (ashish10677) [inline README.md:141]: Did you try just `--autoVote` not `--autoVote true`. That should also work if I'm not wrong.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Yashk767) [reply README.md:141]: yes it also works with just `--autoVote`

Comment (ashish10677) [reply README.md:141]: Ok

**Review by ashish10677 — APPROVED** [review]

**Comment by ashish10677** [comment]
@Yashk767 please resolve conflicts. Also the coverage is going down.

**Comment by Yashk767** [comment]
> @Yashk767 please resolve conflicts. Also the coverage is going down.

Okay will resolve the conflicts. We didn't refactor stake.go yet so the added feature cannot be tested now as the added code is not a part of any function. Will include that in testing by mockery, will that be okay?

**Comment by ashish10677** [comment]
> > @Yashk767 please resolve conflicts. Also the coverage is going down.
> 
> Okay will resolve the conflicts. We didn't refactor stake.go yet so the added feature cannot be tested now as the added code is not a part of any function. Will include that in testing by mockery, will that be okay?

Okay

**Review by ashish10677 — APPROVED** [review]

**Pull request merged**